app/code/Magento/ReviewGraphQl/Model/Resolver/Product/Review/AverageRating.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\ReviewGraphQl\Model\Resolver\Product\Review;
+
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+use Magento\Framework\GraphQl\Query\Resolver\ContextInterface;
+use Magento\Framework\GraphQl\Query\Resolver\Value;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Review\Model\RatingFactory;
+use Magento\Review\Model\Review;
+
+/**
+ * Review average rating resolver
+ */
+class AverageRating implements ResolverInterface
+{
+    /**
+     * @var RatingFactory
+     */
+    private $ratingFactory;
+
+    /**
+     * @param RatingFactory $ratingFactory
+     */
+    public function __construct(
+        RatingFactory $ratingFactory
+    ) {
+        $this->ratingFactory = $ratingFactory;
+    }
+
+    /**
+     * Resolves review average rating
+     *
+     * @param Field $field
+     * @param ContextInterface $context
+     * @param ResolveInfo $info
+     * @param array|null $value
+     * @param array|null $args
+     *
+     * @return float|Value|mixed
+     *
+     * @throws GraphQlInputException
+     *
+     * @SuppressWarnings(PHPMD.UnusedFormalParameter)
+     */
+    public function resolve(
+        Field $field,
+        $context,
+        ResolveInfo $info,
+        array $value = null,
+        array $args = null
+    ) {
+        if (!isset($value['model'])) {
+            throw new GraphQlInputException(__('Value must contain "model" property.'));
+        }
+
+        /** @var Review $review */
+        $review = $value['model'];
+        $summary = $this->ratingFactory->create()->getReviewSummary($review->getId());
+        $averageRating = $summary->getSum() ?: 0;
+
+        if ($averageRating > 0) {
+            $averageRating = (float) number_format($summary->getSum() / $summary->getCount() / 20, 2);
```

Comment:
Hi @eduard13. The average_rating is a percentage value, ranging from 0 to 100. I see you have forced it to fit to a 1 to 5 scale by **dividing it by 20**. This scaling can be avoided since clients can use percentages to fit the ranges. And as you know the rating_summary field is already dealing in percentages and fitting average_rating from 1 to 5 will be inconsistent. 
I have added a picture on how Luma does this using percentages
cc: @paliarush @nrkapoor 
<img width="854" alt="Screen Shot 2020-07-06 at 2 47 34 PM" src="https://user-images.githubusercontent.com/43612864/86636534-03edb400-bf9a-11ea-84e5-3a6be1909d78.png">

Reply:
I've previously raised this question [here](https://github.com/magento/magento2/pull/27882#discussion_r422056754), but as far as I haven't got any answers, just decided to leave it as it was. 
So, thank you for bringing it up, will remove dividing it by 20, so we would have a proper percentage value.